addons/binding/org.openhab.binding.robonect/README.md

```diff
@@ -0,0 +1,123 @@
+# Robonect Binding
+
+Robonect is a piece of hardware which has to be put into your Husqvarna, Gardena and other branded automower and makes 
+it accessable in your internal network. More details about the Robonect module can be found at [robonect.de](http://www.robonect.de)
+
+This binding integrates mowers having the robonect module installed as a thing into the home automation solution, allowing to
+control the mower and react on mower status changes in rules. 
+
+## Supported Things
+
+The binding exposes just one Thing type which is the `mower`.
+
+Tested mowers
+
+| Mower                   | Robonect module  | Robonect firmware version |
+|-------------------------|------------------|---------------------------|
+| Husqvarna Automower 105 | Robonect Hx      | 0.9c, 0.9e                |
+| Husqvarna Automower 315 | Robonect Hx      | 0.9e, 1.0 preview         |
+
+## Discovery
+
+Robonect does not support automatic discovery. So the thing has to be added manually either via Paper UI or things configuration.
+
+## Thing Configuration
+
+following configuration settings are supported for the `mower` thing.
+
+| property name | mandatory | description                                                        |
+|---------------|-----------|--------------------------------------------------------------------|
+| host          | yes       | the hostname or ip address of the mower.                           |
+| pollInterval  | no        | the interval for the binding to poll for mower status information. |
+| offlineTimeout| no        | the maximum time, the mower can be offline before the binding triggers the offlineTrigger channel  |
+| user          | no        | the username if authentication is enabled in the firmware.         |
+| password      | no        | the password if authenticaiton is enabled in the firmware.         |
+
+
+An example things configuration might look like
+
+```
```

Comment:
Please use a specific codefence here.

` ```java `

Reply:
added... However, why java? Shouldn't this be 'text' as well

Reply:
done